docs/tags/partial.md

```diff
@@ -1,6 +1,8 @@
 @function can-stache.tags.partial {{>key}}
 @parent can-stache/deprecated
 
+@deprecated {4.15} Use passing renderer functions through the ViewModel instead.
```

Comment:
Do we have docs anywhere on this? It’d be nice to add a link.

Reply:
@chasenlehara I can not find a link that talks about `enderer functions through the ViewModel instead.`, should we document it?

Reply:
I would change this sentence to say something like:

```
@deprecated {4.15} Pass renderer functions through the ViewModel instead. See the “Calling views” section in the [can-stache#BasicUse can-stache docs] for an example.
```

Reply:
Done!